README.md

```diff
@@ -176,6 +176,245 @@ kardinal manager remove
 kubectl delete ns prod
 ```
 
+## Deploying Kardinal on a Kubernetes Cluster
+
+These instructions provide a guide for deploying Kardinal on any Kubernetes cluster, whether it's a local setup like Minikube, a managed cloud service, or your own self-hosted cluster. We'll use kubectl port-forwarding to access the services, which works universally across different Kubernetes setups.
+
+### Prerequisites
+
+- A Kubernetes cluster (e.g., Minikube, EKS, GKE, AKS, or any other Kubernetes distribution)
+- kubectl installed and configured to access your cluster
+
+### Steps
+
+1. Install the Kardinal CLI:
+
+```bash
+curl https://raw.githubusercontent.com/kurtosis-tech/kardinal/main/scripts/install_cli.sh -s | sh
+```
+
+2. Install Istio (if not already installed):
+
+If you don't already have Istio installed in your cluster, follow these steps to install it:
+
+```bash
+curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.22.1 TARGET_ARCH=x86_64 sh -
+cd istio-1.22.1
+export PATH=$PWD/bin:$PATH
+echo 'export PATH=$PATH:'"$PWD/bin" >> ~/.bashrc
+istioctl install --set profile=demo -y
+cd ..
+```
+
+If you already have Istio installed, you can skip this step.
+
+3. Deploy the Kardinal Manager:
+
+```bash
+kardinal manager deploy kloud-kontrol
+```
+
+4. Note the tenant UUID generated during this process. You'll need this to check your traffic configuration.
+
+5. Clone the Kardinal Playground repository to get the voting app demo:
+
+```bash
+git clone https://github.com/kurtosis-tech/kardinal-playground.git
+cd kardinal-playground/voting-app-demo
+```
+
+6. Deploy the voting-app application with Kardinal:
+
+```bash
+kardinal deploy --docker-compose docker-compose.yaml
+```
+
+7. Check the initial Kardinal traffic configuration:
+   Visit https://app.kardinal.dev/{your-tenant-id} (replace {your-tenant-id} with the UUID from step 4)
+   You should see only the production version of your application in the traffic configuration.
+
+8. Use the following script to set up port-forwarding for accessing the services. Save this script as `kardinal-port-forward.sh`:
```

Comment:
So we need to inline the script here? If this is really needed, maybe we could move it to the script folder and do a curl | sh call no?

Reply:
addressed!